src/services/userServices.js

```diff
@@ -20,3 +20,10 @@ export const updateUserInfo = credentials => {
     .then(res => Promise.resolve(res.data))
     .catch(err => Promise.reject(err.response.data.error));
 };
+
+export const validateToken = credentials => {
+  return Api()
+    .get(`api/v1/user/${credentials.id}`)
```

Comment:
I like this idea of validating the token, but I am wondering if it has to be tightly coupled to the `getting a user` api call. Can you make it stand on its own? A separate route altogether?

Reply:
Okay done, working on the PR for backend.